src/compute/app_runner.rs

```diff
@@ -548,15 +555,23 @@ mod tests {
 
     #[test]
     fn send_computed_file_fails_when_get_challenge_fails() {
-        let runner = DefaultPostComputeRunner::new();
-        let computed_file = create_test_computed_file(Some(TEST_TASK_ID.to_string()));
-        let result = runner.send_computed_file(&computed_file);
+        with_vars(
+            vec![(
+                TeeSessionEnvironmentVariable::SignWorkerAddress.name(),
+                None::<&str>,
+            )],
```

Comment:
Is this a test fix ? Do not hesitate to fix those in dedicated PRs to avoid mixing subjects